scylla/src/client/session.rs

```diff
@@ -1259,15 +1259,9 @@ impl Session {
         async fn prepare_on_all(
             statement: &Statement,
             cluster_state: &ClusterState,
-            on_all_shards: bool,
+            working_connections: &mut (dyn Iterator<Item = Arc<Connection>> + Send),
```

Comment:
Commit: session: prepare_on_all() uses dyn Iterator

Ah, I didn't know about existance of this commit when writing my comment about dyn Iterator.